scaaml/capture/scope/ps6424e.py

```diff
@@ -540,7 +549,7 @@ def _set_channels(self):
             ps.ps6000aSetChannelOn(
                 self.ps_handle,  # handle
                 self.trace.ps_api_channel,  # channel
-                self.trace._coupling,  # coupling
+                self.trace._coupling,  # coupling  # pylint: disable=W0212
```

Comment:
Instead of disabling the warning, what about adding a `coupling_picoapi` or `coupling_internal` get property?